packages/fast-usdc/src/exos/status-manager.js

```diff
@@ -247,6 +244,43 @@ export const prepareStatusManager = (
         );
       },
 
+      /**
+       * If minted while advancing, publish a status update for the advance
+       * to vstorage.
+       *
+       * Does not add or amend `pendingSettleTxs` as this has
+       * already settled.
+       *
+       * @param {EvmHash} txHash
+       * @param {boolean} success whether the Transfer succeeded
+       */
+      advanceOutcomeForSettled(txHash, success) {
```

Comment:
>I liked notify since all we're doing here is a vstorage write.

That shouldn't be a commitment to the caller. The caller tells the status manager about stuff and the status manager abstracts what it does with that information.

> Technically, it's not settled yet (we could say minted).

Yeah that would be better.

>Where does that leave us with notifyObserved?

To be renamed for the same reasons

Reply:
I landed on `advanceOutcomeForMintedEarly` and `advanceOutcomeForUnknownMint`. If we want something more concise, i think we can tackle that in https://github.com/Agoric/agoric-sdk/issues/10432